### PR TITLE
En 1442 my profile

### PR DESCRIPTION
**Pull request by nicpuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/users/my-profile/MyProfilePage.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import { FormattedMessage } from 'react-intl';
+import { Grid, Row, Col, Breadcrumb } from 'patternfly-react';
+import { BreadcrumbItem } from 'frontend-common-components';
+
+import InternalPage from 'ui/internal-page/InternalPage';
+import PageTitle from 'ui/internal-page/PageTitle';
+import ErrorsAlertContainer from 'ui/common/form/ErrorsAlertContainer';
+import PasswordFormContainer from 'ui/users/my-profile/PasswordFormContainer';
+
+const AddUserPage = () => (
```
Comment (alessioscalici) [inline src/ui/users/my-profile/MyProfilePage.js]: Why not `MyProfilePage`?

Comment (nicpuddu) [reply src/ui/users/my-profile/MyProfilePage.js]: my bad! fixed it

**Comment by entando-service** [comment]
All is well

**Pull request merged**